sbtix-tool.nix

```diff
@@ -44,6 +44,20 @@ let
       mkdir -p "$SBTIX_GLBASE_DIR/plugins"
     fi
 
+    SBTIX_SETTINGS_FILE="$SBTIX_GLBASE_DIR/sbtix_settings.sbt"
```

Comment:
A temporary file would be preferable. This solution makes concurrent use of sbtix impossible (assuming it wasn't already; not sure).

Reply:
Good point. I can imagine situations where I want to use `sbtix` on multiple projects without waiting for one another.